References/Control Coverage/Readme.md

```diff
@@ -20,14 +20,16 @@ Below resource types can be checked for validating the security controls.
 <table>
```

Comment:
to a potentially malicious (in Disable request access policy in your organization)